### ISSUE TITLE
Site is down

### ISSUE THREAD

**Issue by Hassuunna** [jslatts/nodechat-tutorial#4]
sorry to open this as an issue but site you mention for tutorials is down, 
Can you please host them on another site or **upload them to YouTube**?

**Comment by jslatts** [comment]
@Hassuunna Thanks for your interest. I would **strongly** encourage you to find a more up to date resource. This tutorial was written when node was very new and everything about it is out of date. I think it will be hard to follow and get working.

If you really want to see the original posts, you can still find them on the wayback machine (http://web.archive.org/web/20160304021337/http://fzysqr.com/2011/02/28/nodechat-js-using-node-js-backbone-js-socket-io-and-redis-to-make-a-real-time-chat-app/#more-400).
